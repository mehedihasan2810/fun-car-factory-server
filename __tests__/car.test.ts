import { testServer } from ".";

describe("Car Queries", () => {
  it("checking jest", async () => {
    const response = await testServer.executeOperation({
      query: `#graphql 
      query GetCars {
           getCars {
              name
           } 
          }`,
      // variables: { name: "world" },
    });

    expect(response).toMatchSnapshot();
  });
});
