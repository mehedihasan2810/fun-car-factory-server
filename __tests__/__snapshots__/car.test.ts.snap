// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Car Queries checking jest 1`] = `
{
  "body": {
    "kind": "single",
    "singleResult": {
      "data": {
        "getCars": [
          {
            "name": "Ferrari 2021",
          },
          {
            "name": "Tesla Super Truck",
          },
          {
            "name": "Tesla Super Truck",
          },
          {
            "name": "TaTa super bus",
          },
          {
            "name": "TaTa super bus",
          },
          {
            "name": "Toyota truck 2011",
          },
          {
            "name": "Volbo bus",
          },
          {
            "name": "Volbo bus",
          },
          {
            "name": "Volbo bus",
          },
          {
            "name": "Volbo bus",
          },
          {
            "name": "Volbo bus",
          },
          {
            "name": "Volbo bus",
          },
          {
            "name": "Volbo bus",
          },
          {
            "name": "Volbo bus",
          },
          {
            "name": "American ferrari",
          },
          {
            "name": "American ferrari",
          },
          {
            "name": "American ferrari",
          },
          {
            "name": "American ferrari",
          },
          {
            "name": "American ferrari",
          },
          {
            "name": "American ferrari",
          },
          {
            "name": "super ferrari",
          },
          {
            "name": "super ferrari",
          },
          {
            "name": "super ferrari",
          },
          {
            "name": "super ferrari",
          },
          {
            "name": "super ferrari",
          },
          {
            "name": "kolkatar taxi",
          },
          {
            "name": "deshi taxi",
          },
          {
            "name": "TaTa Taxi",
          },
          {
            "name": "TaTa Taxi",
          },
          {
            "name": "TaTa Taxi",
          },
        ],
      },
      "errors": undefined,
    },
  },
  "http": {
    "headers": Map {
      "cache-control" => "no-store",
    },
    "status": undefined,
  },
}
`;
